src/promise-katas-1.js

```diff
@@ -33,71 +33,210 @@ Remember, a Promise can only resolve successfully, or reject when it fails to re
 // when we call returnPromise like 'returnPromise()' we expect our function to use the
 // 'resolve' Promise callback function in order to resolve successfully
 
-const returnPromise = () => { };
+const returnPromise = () => { 
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve();
+    }, 1000);
```

Comment:
You are not using reject here, so you dont have to pass it in.
Also Im not sure why the setTime out is. 
You can just return a one liner:
````
return new Promise((resolve) => resolve());
`````